src/styles/blocks/day.scss

```diff
@@ -0,0 +1,26 @@
+.calendar__day {
+  width: $day-size;
+  height: $day-size;
+  border: $border-size solid black;
+  background-color: $grey-color;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  transition:
+    transform $animation-duration ease,
```

Comment:
Ensure that the `$animation-duration` variable is defined. If not, this could lead to a compilation error or unexpected styling results.